src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.NativeAot.cs

```diff
@@ -361,6 +361,27 @@ public static void PrepareDelegate(Delegate d)
 
             return RuntimeImports.RhNewObject(mt);
         }
+
+        public static unsafe int SizeOf(RuntimeTypeHandle type)
+        {
+            if (type.IsNull)
+                ThrowHelper.ThrowArgumentNullException(ExceptionArgument.type);
+
+            MethodTable* mt = type.ToMethodTable();
+
+            if (mt->ElementType == EETypeElementType.Void
+                || mt->ContainsGenericParameters)
```

Comment:
```suggestion
                || mt->IsGenericTypeDefinition)
```
I expect that this is going to fail the build. Native AOT does not materialize MethodTables for these abstract types, and so it does not have `ContainsGenericParameters` property.